src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java

```diff
@@ -55,7 +55,7 @@
   private static final String WIKIA_MESSAGE_BUBLE = "#WikiaNotifications div[id*='msg']";
   private static final String LIFTIUM_IFRAME_SELECTOR = "iframe[id*='Liftium']";
   private static final String GPT_DIV_SELECTOR = "[data-gpt-creative-size]";
-  private static final String TOP_INCONTENT_BOXAD_SELECTOR = "div[id*='TOP_INCONTENT_BOXAD']";
+  private static final String INCONTENT_BOXAD_SELECTOR = "div[id*='INCONTENT_1C']";
```

Comment:
id*='INCONTENT_1'

(can be any of the INCONTENT_1A, INCONTENT_1B, INCONTENT_1C)